torch/_dynamo/variables/builder.py

```diff
@@ -1562,22 +1562,32 @@ def __eq__(self, other: object) -> bool:
 
 # Performs automatic dynamic dim determination.
 # Returns a CreateSymbolicPolicy
-def _automatic_dynamic(e, tx, name, static_shapes) -> CreateSymbolicPolicy:
+def _automatic_dynamic(e, tx, source, static_shapes) -> CreateSymbolicPolicy:
+    name = source.name()
+    prior_policy = tx.output.tracing_context.tensor_to_policy.get(e, None)
+    source_to_symint_node_cache = (
+        prior_policy.source_to_symint_node_cache if prior_policy else None
+    )
```

Comment:
Here, it should be impossible for there to a prior policy, right?

Reply:
There are cases where we can get priors, we don't dedupe params as they are not lifted.